packages/core/src/form-field/FormField.tsx

```diff
@@ -1,14 +1,13 @@
 import { clsx } from "clsx";
 import { ForwardedRef, forwardRef, HTMLAttributes } from "react";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
 import { A11yValueProps, FormFieldContext } from "../form-field-context";
 import { makePrefixer, useId, capitalize } from "../utils";
 
-import { useWindow } from "@salt-ds/window";
-import { useComponentCssInjection } from "@salt-ds/styles";
-
 import formFieldCss from "./FormField.css";
 
-export type FormFieldLabelPlacement = "top" | "left";
+export type FormFieldLabelPlacement = "top" | "left" | "right" | undefined;
```

Comment:
`undefined` shouldn't be here as it is not a valid label placement.  It should be specified by the *prop* which has the type `FormFieldLabelPlacement`